pkg/eve/Dockerfile.in

```diff
@@ -2,6 +2,8 @@ FROM lfedge/eve-alpine:145f062a40639b6c65efa36bed1c5614b873be52 as tools
 ENV PKGS qemu-img tar u-boot-tools coreutils dosfstools
 RUN eve-alpine-deploy.sh
 
+# hadolint ignore=DL3006
+FROM BSP_IMX_TAG as bsp-imx
```

Comment:
Please add `pkg/bsp-imx` to [PKGS_riscv64](https://github.com/lf-edge/eve/blob/eb862d9da859a293f08ae2845f3dd724c35649e4/Makefile#L301-L303).

Reply:
Yes, thanks, I saw that the build on riscv64 was falling.